Assets/Plugins/Editor/Uplift/UpliftPreferences.cs

```diff
@@ -60,10 +67,27 @@ public static void PreferencesGUI()
             );
             useExperimentalFeatures = EditorGUILayout.Toggle("Use experimental features", useExperimentalFeatures);
 
-            if(GUI.changed)
+            EditorGUILayout.LabelField("Github related:", EditorStyles.boldLabel);
+            EditorGUILayout.HelpBox(
+                "Github does not support TLS versions 1.0 and 1.1 as of 22/02/2018, and TLS 1.2 is available in Unity only with Mono 4.8. If you want to use a Github Repository, it will need to be behind a proxy",
+                MessageType.Info
+            );
+            useGithubProxy = EditorGUILayout.Toggle("Use experimental features", useGithubProxy);
+            GUI.enabled = useGithubProxy;
+            if (useGithubProxy)
+                EditorGUILayout.HelpBox(
+                    "The proxy URL will replace 'https://api.github.com'",
+                    MessageType.Info
+                );
+            githubProxyUrl = EditorGUILayout.TextField("Proxy URL", githubProxyUrl);
```

Comment:
we could make sure that the specified values does not end with `/` (or remove it if there's any).